src/filehandler/pemreader.cpp

```diff
@@ -39,29 +39,20 @@ void PemFileReader::checkFile() const
 
 Address PemFileReader::getAddress() const
 {
-    return Address(bytes(m_fileKeyBytes.begin() + crypto_sign_PUBLICKEYBYTES,m_fileKeyBytes.end()));
+    bytes const pubKey(m_fileKeyBytes.begin() + SEED_BYTES_LENGTH, m_fileKeyBytes.end());
+    return Address(pubKey);
 }
 
 bytes PemFileReader::getSeed() const
 {
-    return bytes(m_fileKeyBytes.begin(),m_fileKeyBytes.begin() + crypto_sign_PUBLICKEYBYTES);
+    return bytes(m_fileKeyBytes.begin(), m_fileKeyBytes.begin() + SEED_BYTES_LENGTH);
 }
 
-bytes PemFileReader::getPrivateKey() const
+bytes PemFileReader::getSecretKey() const
```

Comment:
@AdoAdoAdo, should we keep both `getSeed()` and `getSecretKey()`, here, in the PEM reader? Or remove the latter - seems not be used, right?

Reply:
I would remove the later if not directly used, so at least we avoid confusion of which is the secret key and which is the seed.

Reply:
Agree, I removed it, as well as other unused functions and one test